contentcuration/contentcuration/serializers.py

```diff
@@ -149,7 +122,7 @@ class FileSerializer(BulkSerializerMixin, serializers.ModelSerializer):
     language = LanguageSerializer(many=False, required=False, allow_null=True)
     display_name = serializers.SerializerMethodField('retrieve_display_name')
     id = serializers.CharField(required=False)
-    preset = FormatPresetSerializer(many=False)
+    preset = FormatPresetSerializer(many=False, read_only=True)
```

Comment:
make sure we're not trying to save a new FormatPreset type

Reply:
Try removing this and see if it still works. I think the real problem was fixed when you removed the code to set the file preset above.

Reply:
Yup... It's as you suspected!